controllers/state_of_the_world.go

```diff
@@ -404,6 +447,7 @@ func (b *BootOptionsBuilder) Reconciler() controller.ReconcileFunc {
 			NewTLSWorkflow(b.client, b.manager.GetScheme(), b.isGatewayAPIInstalled, b.isCertManagerInstalled).Run,
 			NewDataPlanePoliciesWorkflow(b.client, b.isGatewayAPIInstalled, b.isIstioInstalled, b.isEnvoyGatewayInstalled, b.isLimitadorOperatorInstalled, b.isAuthorinoOperatorInstalled).Run,
 			NewKuadrantStatusUpdater(b.client, b.isGatewayAPIInstalled, b.isGatewayProviderInstalled(), b.isLimitadorOperatorInstalled, b.isAuthorinoOperatorInstalled).Subscription().Reconcile,
+			NewObservabilityReconciler(b.client, b.manager.GetRESTMapper(), operatorNamespace).Subscription().Reconcile,
```

Comment:
I have being question my self why only the observability reconciler requires the `GetRESTMapper`, while no other reconcile does. It seems like the only place that it is required is setting up for the `r.Client.Resource`, and that is done in other reconcilers also. So like in a previous comment I am wondering would the better thing to do be set up vars like there is in `pkg/istio/utils.go`. This I believe would remove the dependency on the `GetRESTMapper`.

Reply:
I'm not completely following what you're suggesting.
The code in observability_reconciler.go that relies on the restmapper is used to create a monitor (Pod or Service), and is generic in that it needs the restmapping capability to set up the resource client based on the kind.
Are you suggesting something in utils that sets up those clients ahead of time?

Reply:
> My biggest concern is the use of the GetRESTMapper
What is the concern with using it?

Reply:
Discussed offline, the result of which is a refactor of the schemas out to pkg/observability/schemas.go file

Reply:
I was expecting to see the `GetRESTMapper` removed, but with how the delete all is implemented it makes sense why it is still there. I have no issue with it being there, only a little surprised that's all.